packages/react-tinacms-editor/src/components/Wysiwyg/index.tsx

```diff
@@ -57,16 +57,15 @@ export const RawModeEditor = ({
           toggleEditorMode={() => setMode('wysiwyg')}
         />
       ) : (
-        <WysiwygEditor
+        <ProsemirrorEditor
           input={{
             value,
             onChange: handleChange,
           }}
```

Comment:
Do you remember the reason for the indirection of the `value` and `onChange`?

I'm pretty sure that keeping the value in state will introduce the bug. 

```suggestion
          input={input}
```

The editor should not be keeping it's own state, that's the job of the form object outside of it. By keeping track of the value here the editor component can get out of sync with the form's data.

Reply:
Recent values is needed here so that it can be kept in sync between markdown and wysiwyg editors.

Values here is not creating issues, reason being that editors in library behave as un-controlled components. Making editors controlled can be bit expensive as it requires conversion from markdown to prosemirror json for each change in content.